plugins/calendar/.settings/org.eclipse.jdt.core.prefs

```diff
@@ -20,7 +20,7 @@ org.eclipse.jdt.core.compiler.doc.comment.support=enabled
 org.eclipse.jdt.core.compiler.problem.APILeak=warning
 org.eclipse.jdt.core.compiler.problem.annotationSuperInterface=warning
 org.eclipse.jdt.core.compiler.problem.assertIdentifier=error
-org.eclipse.jdt.core.compiler.problem.autoboxing=warning
+org.eclipse.jdt.core.compiler.problem.autoboxing=ignore
```

Comment:
this file should not be committed

Reply:
why? this is part of the code conventions

Reply:
ah i forgot about that, nvm!